tests/forms/Droid/MainActivity.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+using Android.App;
+using Android.Content;
+using Android.Content.PM;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using Android.OS;
+
+namespace forms.Droid
+{
+    [Activity(Label = "forms.Droid", Icon = "@drawable/icon", Theme = "@style/MyTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
+    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    {
+        bool firstOnCreate = true;
+		bool firstOnStart = true;
+		bool firstOnResume = true;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            if (firstOnCreate)
+                Console.WriteLine("startup-timing: OnCreate reached");
+            TabLayoutResource = Resource.Layout.Tabbar;
+            ToolbarResource = Resource.Layout.Toolbar;
+
+            base.OnCreate(bundle);
+
+            global::Xamarin.Forms.Forms.Init(this, bundle);
+
+            LoadApplication(new App());
+            if (firstOnCreate) {
+                Console.WriteLine("startup-timing: OnCreate end reached");
+                firstOnCreate = false;
+            }
+		}
+
+        protected override void OnActivityResult(int requestCode, Result resultCode, Intent data)
+        {
+            base.OnActivityResult(requestCode, resultCode, data);
+        }
+
+        protected override void OnStart()
+        {
+            if (firstOnStart)
+                Console.WriteLine("startup-timing: OnStart reached");
+			base.OnStart();
+            if (firstOnStart) {
+                Console.WriteLine("startup-timing: OnStart end reached");
+                firstOnStart = false;
+            }
+		}
```

Comment:
This line is indented too much; it should be aligned with the `{` for `OnStart()`.